db/schema.rb

```diff
@@ -11,12 +11,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160128233820) do
+ActiveRecord::Schema.define(version: 20160129095819) do
 
   create_table "answers", force: :cascade do |t|
     t.integer  "user_id"
     t.integer  "question_id"
     t.string   "content"
+    t.integer  "votes"
```

Comment:
sure thing, That's probably from the last merge: I'm sure we've not added a migration to remove the votes from either answers/questions/comments.